service_load_balancing/variables.tf

```diff
@@ -57,41 +71,37 @@ variable "iam_role_inline_policy" {
 EOT
 }
 
-variable "placement_strategy_type" {
-  description = "The type of placement strategy"
-  default     = "spread" // or binpack (this module cannot specify "random"
-}
 
-variable "placement_strategy_field" {
-  description = "The field of placement strategy valid values that U select type"
-  default     = "instanceId"
+# ECS launch_type: FARGATE
+
+variable "subnet_ids" {
+  description = "AWS vpc zone identifier(s)"
+  type        = "list"
+  default     = []
 }
 
-variable "desired_count" {
-  description = "The number of instances of the task definition"
-  default     = 0
+variable "security_group_ids" {
+  description = "AWS security group id(s) for container instances launch configuration"
+  type        = "list"
+  default     = []
 }
 
-variable "deployment_maximum_percent" {
-  description = "The upper limit of the number of running tasks that can be running in a service during a deployment"
-  default     = 200
+# ECS launch_type: EC2
+
+variable "cluster_id" {
+  description = "ID (ARN) of ECS Cluster if created by external"
+  default     = ""
```

Comment:
`aws_ecs_service.cluster` is optional (just note)
https://www.terraform.io/docs/providers/aws/r/ecs_service.html#cluster